lib/core-integration/src/Test/Integration/Framework/DSL.hs

```diff
@@ -1047,6 +1053,12 @@ updateWalletPassEp w =
 --- CLI
 ---
 
+runJcli
+    :: CmdResult r
+    => [String]
+    -> IO r
+runJcli = command [] "jcli"
```

Comment:
does CI have access to jcli ?

Reply:
Yes, it's installed along with `jormungandr`.